lib/elixir_sense/providers/suggestion.ex

```diff
@@ -277,20 +279,42 @@ defmodule ElixirSense.Providers.Suggestion do
     end
   end
 
-  @spec find_hint_mods_funcs(String.t(), [module], [{module, module}], module, %{}) :: %{
-          hint: hint,
-          suggestions: [mod | func]
-        }
-  defp find_hint_mods_funcs(hint, imports, aliases, module, mods_and_funs) do
+  @spec find_hint_mods_funcs(String.t(), [module], [{module, module}], module, %{}, String.t()) ::
+          %{
+            hint: hint,
+            suggestions: [mod | func]
+          }
+  defp find_hint_mods_funcs(hint, imports, aliases, module, mods_and_funs, text_before) do
     env = %Complete.Env{
       aliases: aliases,
       scope_module: module,
       imports: imports,
       mods_and_funs: mods_and_funs
     }
 
-    {hint_suggestion, suggestions} = Complete.run(hint, env)
-    %{hint: hint_suggestion, suggestions: suggestions}
+    {hint, prefix} =
+      case Source.get_v12_module_prefix(text_before) do
+        nil ->
+          {hint, ""}
+
+        module ->
+          # v1.2 alias syntax detected
+          # prepend module prefix before running completion
+          prefix = (module |> Atom.to_string() |> String.replace("Elixir.", "")) <> "."
```

Comment:
I believe `(module |> Atom.to_string() |> String.replace("Elixir.", ""))` could be replaced by `inspect(module)`

Reply:
+1